src/uagents/agent.py

```diff
@@ -423,9 +426,14 @@ async def _process_message_queue(self):
 
 
 class Bureau:
-    def __init__(self, port: Optional[int] = None):
+    def __init__(
+        self,
+        port: Optional[int] = None,
+        endpoint: Optional[Union[List[str], Dict[str, dict]]] = None,
```

Comment:
```suggestion
        endpoint: Optional[Union[str, List[str], Dict[str, dict]]] = None,
```
To match your other change